File: chatwoot/csharp/src/OSEG.ChatwootExamples/GetAccountAutomationRuleExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Com.Chatwoot.Api;
using Com.Chatwoot.Client;
using Com.Chatwoot.Model;

namespace OSEG.ChatwootExamples;

public class GetAccountAutomationRuleExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("api_access_token", "USER_API_KEY");

        try
        {
            var response = new AutomationRuleApi(config).GetAccountAutomationRule(
                accountId: 0,
                page: 1
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling AutomationRuleApi#GetAccountAutomationRule: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: chatwoot/typescript-axios/src/GetAccountAutomationRuleExample.ts
import axios, { AxiosError } from "axios";
import * as api from "chatwoot_client"

const configuration = new api.Configuration({
  apiKey: "USER_API_KEY",
});

new api.AutomationRuleApi(configuration).getAccountAutomationRule(
  0, // accountId
  1, // page
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling AutomationRuleApi#getAccountAutomationRule:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
